Backpack/Nudger/README.md

```diff
@@ -0,0 +1,47 @@
+# Backpack/Nudger
+
+## Usage
+
+`BPKNudger` contains the Backpack Nudger component. It's an adjustable control that allows a traveller to select a numerical value between a min/max determined by the consumer.
+
+It uses the iOS accessibility adjustable trait so that users of Assistive Technology can easily interact with it.
+
+### Objective-C
+
+```objective-c
+#import <Backpack/Nudger.h>
+
+BPKNudgerConfiguration *nudgerConfiguration = [[BPKNudgerConfiguration alloc]
+    initWithLabelFormatter:^(double value) {
+        NSNumber *number = [NSNumber numberWithDouble:value];
+        return [NSNumberFormatter localizedStringFromNumber:number numberStyle:NSNumberFormatterDecimalStyle];
+    }
+    accessibilityLabelFormatter:^(double value) {
+        return [NSString stringWithFormat:@"%@ passengers", [NSNumber numberWithDouble:value]];
```

Comment:
This should also use `localizedStringFromNumber:`?

Reply:
We might even wanna showcase how using NSLocalizedString with pluralisation support is the appropriate solution here?

Reply:
I'm wary of making the readme too complicated. They're hard to maintain.

Reply:
Fair point